Dockerfile

```diff
@@ -0,0 +1,53 @@
+# Copyright Contributors to the Open Cluster Management project
+
+FROM registry.ci.openshift.org/open-cluster-management/builder:go1.16-linux AS builder
+
+WORKDIR /go/src/github.com/open-cluster-management/search-api
```

Comment:
Should this be search-v2-api? Also, on line 46.

Reply:
search-v2-api = yes , Will fix that @SherinV